src/Http/Routing/perf/Matching/MatcherGithubBenchmarkBase.generated.cs

```diff
@@ -14,1226 +14,1227 @@ public partial class MatcherGithubBenchmarkBase : EndpointRoutingBenchmarkBase
-            Endpoints[214] = CreateEndpoint("/repos/{owner}/{repo}/pulls/{number}", "PATCH");
-            Endpoints[215] = CreateEndpoint("/repos/{owner}/{repo}/releases/{id}", "PATCH");
-            Endpoints[216] = CreateEndpoint("/repos/{owner}/{repo}/releases/{id}", "GET");
-            Endpoints[217] = CreateEndpoint("/repos/{owner}/{repo}/releases/{id}", "DELETE");
-            Endpoints[218] = CreateEndpoint("/repos/{owner}/{repo}/statuses/{ref}", "GET");
-            Endpoints[219] = CreateEndpoint("/repos/{owner}/{repo}/statuses/{ref}", "POST");
-            Endpoints[220] = CreateEndpoint("/repos/{owner}/{repo}/commits/{ref}/status", "GET");
-            Endpoints[221] = CreateEndpoint("/repos/{owner}/{repo}/commits/{shaCode}/comments", "GET");
-            Endpoints[222] = CreateEndpoint("/repos/{owner}/{repo}/commits/{shaCode}/comments", "POST");
-            Endpoints[223] = CreateEndpoint("/repos/{owner}/{repo}/deployments/{id}/statuses", "GET");
-            Endpoints[224] = CreateEndpoint("/repos/{owner}/{repo}/deployments/{id}/statuses", "POST");
-            Endpoints[225] = CreateEndpoint("/repos/{owner}/{repo}/hooks/{hookId}/tests", "POST");
-            Endpoints[226] = CreateEndpoint("/repos/{owner}/{repo}/issues/{number}/comments", "POST");
-            Endpoints[227] = CreateEndpoint("/repos/{owner}/{repo}/issues/{number}/comments", "GET");
-            Endpoints[228] = CreateEndpoint("/repos/{owner}/{repo}/issues/{number}/events", "GET");
-            Endpoints[229] = CreateEndpoint("/repos/{owner}/{repo}/issues/{number}/labels", "POST");
-            Endpoints[230] = CreateEndpoint("/repos/{owner}/{repo}/issues/{number}/labels", "GET");
-            Endpoints[231] = CreateEndpoint("/repos/{owner}/{repo}/issues/{number}/labels", "PUT");
-            Endpoints[232] = CreateEndpoint("/repos/{owner}/{repo}/issues/{number}/labels", "DELETE");
-            Endpoints[233] = CreateEndpoint("/repos/{owner}/{repo}/milestones/{number}/labels", "GET");
-            Endpoints[234] = CreateEndpoint("/repos/{owner}/{repo}/pulls/{number}/comments", "GET");
-            Endpoints[235] = CreateEndpoint("/repos/{owner}/{repo}/pulls/{number}/comments", "POST");
-            Endpoints[236] = CreateEndpoint("/repos/{owner}/{repo}/pulls/{number}/commits", "GET");
-            Endpoints[237] = CreateEndpoint("/repos/{owner}/{repo}/pulls/{number}/files", "GET");
-            Endpoints[238] = CreateEndpoint("/repos/{owner}/{repo}/pulls/{number}/merge", "PUT");
-            Endpoints[239] = CreateEndpoint("/repos/{owner}/{repo}/pulls/{number}/merge", "GET");
-            Endpoints[240] = CreateEndpoint("/repos/{owner}/{repo}/releases/{id}/assets", "GET");
-            Endpoints[241] = CreateEndpoint("/repos/{owner}/{repo}/issues/{number}/labels/{name}", "DELETE");
-            Endpoints[242] = CreateEndpoint("/repos/{owner}/{repo}/{archive_format}/{path}", "GET");
+            Endpoints[0] = CreateEndpoint("/emojis", HttpMethods.Get);
```

Comment:
Should I also change the Template for Swaggatherer?

Reply:
> Should I also change the Template for Swaggatherer?

That would be nice.

Do we need these changes at all? Wouldn't the metadata take care of this for you?

If you're just doing this for consistency/cleanliness, I woulnd't worry about it.

Reply:
Oh! you're right. I should have only changed this `Requests[0].Request.Method = HttpMethods.Get;`